tests/frontend/test_kibana_app/cypress/support/index.js

```diff
@@ -23,7 +23,8 @@ import {
     navigate,
     validateURLIncludes,
     setCookies,
-    preserveCookie
+    preserveCookie,
+    timestampToDate
```

Comment:
As we mention before we want to have criteria of how we write the imports, I mean, on one line or in multiple lines?

Reply:
Also the empty line at the end of the file